src/backend/distributed/executor/adaptive_executor.c

```diff
@@ -2530,6 +2536,18 @@ PopUnassignedPlacementExecution(WorkerPool *workerPool)
 }
 
 
+/*
+ * StartPlacementExecutionOnSession gets a TaskPlacementExecition and
+ * WorkerSession, and as the name suggests, the task's query is sent
```

Comment:
I think the "and as the name suggests" is more redundant than just repeating the name in words :)